VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -39,6 +39,7 @@
             <LinearLayout
                 android:id="@+id/last_access_bar"
                 android:orientation="vertical"
+                android:layout_height="wrap_content"
```

Comment:
Although, that style is only used in one place: right here, so we should probably just inline the remaining styles `layout_width` and `background`